src/euphorie/client/browser/risk.py

```diff
@@ -503,11 +503,20 @@ def calculatePriority(self, risk, reply):
 
 class ImageUpload(BrowserView):
 
+    def _return(self):
+        return self.request.response.redirect(
+            "{}/@@identification".format(self.context.absolute_url())
+        )
+
     def __call__(self):
         if self.request.form.get("image"):
             image = self.request.form["image"]
             new_data = image.read()
             if self.context.image_data != new_data:
+                try:
+                    PIL.Image.open(BytesIO(new_data))
```

Comment:
Don't we want to inform the user about what just happened?

Reply:
Yeah, I thought about that too. As a MVP I think it might be good enough to simply not display anything. If we emit a message like "Not a valid image", we need translation again for 20 languages.
But probably, it is better to display a warning. I'll add something.